spec/check_localization_progress_spec.rb

```diff
@@ -0,0 +1,337 @@
+require 'spec_helper.rb'
+require 'webmock/rspec'
+
+describe Fastlane::Actions::CheckTranslationProgressAction do
+  before(:each) do
+    allow(FastlaneCore::UI).to receive(:'message')
```

Comment:
Not sure why Danger is not commenting here, but Rubocop locally gives me this error

```
Style/SymbolLiteral: Do not use strings for word-like symbol literals.
```

In fact, there are a few violations that I get when running `bundle exec rubocop spec/check_localization_progress_spec.rb` locally (I had to rebase on `develop` and run `bundle install` first to workaround an configuration error). 🤔

Reply:
Looking [in CI](https://github.com/wordpress-mobile/release-toolkit/pull/263/checks?check_run_id=2656810727), I think the reason is because these are new files?

This is the output of the Danger run, which also includes the error I mentioned:

```
Updated 0 paths from the index
.rubocop.yml: Metrics/LineLength has the wrong namespace - should be Layout
.rubocop.yml: Metrics/LineLength has the wrong namespace - should be Layout
Error: The `Naming/UncommunicativeMethodParamName` cop has been renamed to `Naming/MethodParameterName`.
(obsolete configuration found in .rubocop_todo.yml, please update it)
```

Weird danger didn't fail the build because of that 🤔 Although it might be due to the inline comment configuration. cc @AliSoftware.

Reply:
The bug with inline comment configuration not always posting comments for all violations is because of a GitHub API limitation (apparently), which likely limits the number of comments per second a bot can post. So when we hit this limitation:
 - We generally at least have N (=2? 3?) Danger comments and only the additional Danger comments that the Danger job should have posted after those first N get dropped. While here we don't have any Danger comment at all
 - To double-check, you can open [the log of the Danger GitHub Action](https://github.com/wordpress-mobile/release-toolkit/pull/263/checks?check_run_id=2656810727) and look at the "Run Danger" step. That step will list all the violations it found, which includes both the first N violations that it would have managed to post an inline comment for and the subsequent violations which the GH limit would have potentially blocked

In our present case this is none of that; instead, Danger got interrupted when rubocop exited with exit code 1 due to a configuration error ("Error: The `Naming/UncommunicativeMethodParamName` cop has been renamed to `Naming/MethodParameterName`.") and didn't do the linting. This was fixed in my other PR (which has now been merged) so rebasing this PR on top of develop should fix that.